### ISSUE TITLE
"Bug" in the documentation: missing displayed code from the package

### ISSUE THREAD

**Issue by quark67** [dgsiegel/latex-tangocolors#3]
In the documentation file (PDF), on page 3, the third column of the table (lines 36 to 44) is not displayed (because the lines are too long).

Perhaps considering another document class file instead of `ltxdoc` can help? On the "Documentation Support" topic from the CTAN (https://ctan.org/topic/doc-supp), I see `ydoc`. On it's documentation (http://mirrors.ctan.org/macros/latex/contrib/ydoc/ydoc.pdf), I see an example (see line 10 on page 8) with a too long line broken in two lines (with another number line). But, there is a red warning at the beginning of the documentation of the `ydoc` document class: **This package bundle is currently under development. (...) It is not ready yet to be used for other packages.**.

But as the documentation of `skdoc` say: *It is loosely based on the `ydoc`*, perhaps `ydoc` can be used despite it's warning.

Or perhaps `doctools` (http://mirrors.ctan.org/macros/latex/contrib/doctools/doctools.pdf). See a long line of code in line 5 of page 7 of the documentation.

Hope this can help you.

**Comment by hpreusse** [comment]
@quark67 I could put that table into landscape mode (as attached). Would this be an option?
[tangocolors.pdf](https://github.com/dgsiegel/latex-tangocolors/files/11082164/tangocolors.pdf)

**Comment by quark67** [comment]
It seems strange with this layout. Another solution can be:

```
\newcommand{\dumptangocolors}{%
  \begingroup
    \renewcommand{\arraystretch}{1.5}
    \begin{tabular}{ccc}
    \cellcolor{butter1}butter1          & \cellcolor{butter2}butter2                          
& \cellcolor{butter3}butter3 \\
    \cellcolor{orange1}orange1          & \cellcolor{orange2}orange2                          
& \cellcolor{orange3}orange3 \\
    \cellcolor{chocolate1}chocolate1    & \cellcolor{chocolate2}chocolate2                    
& \cellcolor{chocolate3}chocolate3 \\
    \cellcolor{chameleon1}chameleon1    & \cellcolor{chameleon2}chameleon2                    
& \cellcolor{chameleon3}chameleon3 \\
    \cellcolor{skyblue1}skyblue1        & \cellcolor{skyblue2}skyblue2                        
& \cellcolor{skyblue3}\textcolor{white}{skyblue3} \\
    \cellcolor{plum1}plum1              & \cellcolor{plum2}plum2                              
& \cellcolor{plum3}\textcolor{white}{plum3} \\
    \cellcolor{scarletred1}scarletred1  & \cellcolor{scarletred2}scarletred2                 
 & \cellcolor{scarletred3}\textcolor{white}{scarletred3} \\
    \cellcolor{aluminium1}aluminium1    & \cellcolor{aluminium2}aluminium2                    
& \cellcolor{aluminium3}aluminium3 \\
    \cellcolor{aluminium4}aluminium4    & \cellcolor{aluminium5}\textcolor{white}{aluminium5} 
& \cellcolor{aluminium6}\textcolor{white}{aluminium6} \\
    \end{tabular}
  \endgroup
}
```

Or, perhaps can the ltxdoc document class accept the landscape layout?

Some packages use the landscape layout for the documentation, e.g. https://ctan.org/pkg/tikz-ext.

**Comment by hpreusse** [comment]
This seems to be the most reasonable solution. I included it in v0.3.
